src/smart-components/workspaces/WorkspaceListTable.tsx

```diff
@@ -107,6 +118,47 @@ const WorkspaceListTable = () => {
     setIsDeleteModalOpen(!isDeleteModalOpen);
   };
 
+  const isValidEditType = (workspace: Workspace) => {
+    switch (workspace.type) {
+      case 'root':
+        return false;
+      case 'default':
+        return true;
+      case 'ungrouped-hosts':
+        return false;
+      case 'standard':
+        return true;
+    }
+  };
+
+  const isValidDeleteType = (workspace: Workspace) => {
+    switch (workspace.type) {
+      case 'root':
+        return false;
+      case 'default':
+        return false;
+      case 'ungrouped-hosts':
+        return false;
+      case 'standard':
+        return true;
+    }
+  };
+
+  const canModify = (workspace: Workspace, action: 'edit' | 'delete') => {
+    if (
+      userPermissions.permission === 'inventory:groups:write' &&
```

Comment:
There is also `inventory:groups:*` permission which allows write. Let's change this
```suggestion
      ['inventory:groups:write', 'inventory:groups:*'].includes(userPermissions.permission) &&
```